### DISCUSSION TITLE
Reset events in database

### DISCUSSION THREAD

**Discussion by irs111** [SimplyStaking/panic#350] [Q&A] [answered]
Hello
How could I reset the the datbase to an empty state, but leaving earlier added cosmos nodes and other configurations behind?
Cheers

**Comment by dillu24** [comment, marked as answer]
Hi @irs111 , at the moment the configurations are not stored on the database, but stored on the filesystem inside the `config` folder. So in other words you could delete the database container. This can be done by doing `sudo docker-compose kill mongo` and `sudo docker-compose rm -v mongo`.

Afterwards restart mongo using `sudo docker-compose up --build -d mongo`

**Reply by irs111** [reply]
Thank you :)
